src/main/resources/application.conf

```diff
@@ -325,7 +325,7 @@ mantis {
 
     # During fast-sync when most up to date block is determined from peers, the actual target block number
     # will be decreased by this value
-    target-block-offset = 128
+    target-block-offset = 32
```

Comment:
it is part of my experiments with fast sync. It seems geth tries to have offset equal to 64 blocks. But their sync is much faster and they process a lot more nodes before updating to new target.
128 is definitly to much, as large part of the peers keeps only 128 blocks history, so it can happen that they won;t have target block root and our sync will not even start.